src/ol/collection.js

```diff
@@ -82,7 +82,7 @@ ol.CollectionProperty = {
  */
 ol.Collection = function(opt_array) {
 
-  goog.base(this);
+  ol.Object.call(this);
```

Comment:
Ok, please disregard my `goog.base` comments above. I was under the impression that `goog.base(this)` (single argument) has a special meaning, which it apparently does not have.